### DISCUSSION TITLE
Update for Marlin 2.1.x

### DISCUSSION THREAD

**Discussion by Tigger777** [FYSETC/FYSETC-E4#44] [General]
Is there any chance of creating a new branch for the E4 from the latest Marlin 2.1.x release.
I've tried and tried, but cannot get comms to the printer from the web interface once built (like the websockets aren't working). There is also no configuration.h and configuration_adv.h files for this board in the marlin config repo.

The reason is that the current E4 branch here does not want to build anymore, and there have been some changes to PlatformIO wrt to 'build_src_filters' which won't be supported soon.

Any help will be most welcome
Duane

**Comment by mx83toy** [comment]
same here can not get this to work

**Reply by TextileTopher** [reply]
Agreed as well. I started manually rebuilding it, but a Marlin refresh would be awesome.

**Reply by mx83toy** [reply]
I've been at it for 2 weeks.. I legit had enough and asked for a refund. I'd rather use my older boards and run octopi on a couple of raspberries than persist with this.
I was excited at the idea of ultra quick esp32, integrated tmc2209's, native WiFi and price... I thought that this board would be so popular, but not if it can't have firmware built.
